stats/opencensus/stats.go

```diff
@@ -122,10 +122,11 @@ func recordRPCData(ctx context.Context, s stats.RPCStats, mi *metricsInfo) {
 		return
 	}
 	switch st := s.(type) {
-	case *stats.InHeader, *stats.OutHeader, *stats.InTrailer, *stats.OutTrailer:
+	case *stats.InHeader, *stats.OutHeader, *stats.InTrailer, *stats.OutTrailer, *stats.PickerUpdated:
 		// Headers and Trailers are not relevant to the measures, as the
 		// measures concern number of messages and bytes for messages. This
 		// aligns with flow control.
+		// Updated pickers are not relevant to the measures, either.
```

Comment:
Nit: please wrap to 80 chars, as above.

Reply:
I would prefer to make the whole comment flow better: "Headers, Trailers, and picker updates are not relevant......"

Reply:
Done.